docs/manual/mkdocs.yml

```diff
@@ -115,7 +115,8 @@ markdown_extensions:
   - pymdownx.inlinehilite
   - pymdownx.keys
   - pymdownx.smartsymbols
-  - pymdownx.snippets
+  - pymdownx.snippets:
+      base_path: 'docs/users/managing_metadata/harvesting/'
```

Comment:
That works if we only use snippets in this one location; for the main manual we may need to use the actual base past. 

I also enjoyed mkdocs-include plugin earlier; so I am curious what we should use long term.